### PR TITLE
DeDuplicate LosingPane and WinPane <organistation, feature

### PR DESCRIPTION
**Pull request by spiderbiggen**

### CODE REVIEW

**Comment by MichielvdBerg** [comment]
Approved.

**Comment by daanwerf** [comment]
Looks nice and clean to me.

File: src/main/java/lasersharksgui/StandardPane.java [hunk 1]
```diff
@@ -40,9 +40,9 @@
   // sprite and image variables
   protected ImageView sharkImage;
 
-  protected static final int SCREEN_POSITION_THREE = 3;
-  protected static final int SCREEN_POSITION_FIVE = 5;
-  protected static final int SCREEN_POSITION_HUNDRED = 100;
+  protected static final int THREE = 3;
```
Comment (Exclaminator) [inline src/main/java/lasersharksgui/StandardPane.java]: What are these constants?

**Comment by Exclaminator** [comment]
Please fix merge conflicts.

**Comment by MichielvdBerg** [comment]
Now the branch is up to dat, it could be merged

**Comment by Exclaminator** [comment]
For nexttime DeDuplication roughly equals Unification

**Pull request merged**